components/nav/nav.css

```diff
@@ -35,14 +35,21 @@
 .header-menu {
   border-left: 3px solid var(--primary);
   border-right: 3px solid var(--primary);
-  padding: 1.4rem;
+  /* padding: 1.4rem; */
 }
 .menu-item {
   display: inline-block;
   font-family: "Roboto", sans-serif;
   font-size: 1rem;
   margin: 0rem 0.7rem;
   color: var(--primary);
+  padding: 1.35rem;
+}
+
+.menu-item:hover{
+  background-color: var(--primary-dark);
+  color:var(--font);
+  cursor: pointer;
 }
 
 a {
```

Comment:
Grabbing an element by name can result in unwanted styles being applied, try fixing this.